_sass/_layout.scss

```diff
@@ -676,22 +718,6 @@ div.facebook {
     margin-top: -20px;
 }
 
-#markdown-toc {
```

Comment:
This breaks the FAQ TOC, but I kind of like it better like that.. ;)

![screen shot 2017-03-15 at 23 18 39](https://cloud.githubusercontent.com/assets/330324/23973290/c782acaa-09d5-11e7-9306-75489ac8554c.png)